src/components/sound/index.tsx

```diff
@@ -111,6 +138,7 @@ export const SoundButton: React.FC<SoundButtonProps> = ({ sound }) => {
         ref={soundRef}
         onCanPlay={() => setSoundState({ ...localSoundState, loaded: true })}
         preload="auto"
+        autoPlay
```

Comment:
With `autoPlay` all sounds starts when enter the page, but without the visual feedback of "active sound".

Why you put this config?

Reply:
I check some forums and sites to solve the problem to play the audio automatically with shareable links, and some of them recommended add autoPlay in the audio tag, but this not solve it, I think I forgot to remove this.